image-index.md

```diff
@@ -43,32 +43,6 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
     This OPTIONAL property describes the platform which the image in the manifest runs on.
```

Comment:
update, what do you think of https://github.com/opencontainers/image-spec/pull/638/commits/3e7b8d00a1c7b5c8561c1fbcbdc8a437f6c82850